src/SDKs/ServiceBus/ServiceBus.Tests/Tests/ScenarioTests.NamespaceTests.Detail.CRUD.cs

```diff
@@ -0,0 +1,183 @@
+                        {
+                            {"tag3", "value3"},
+                            {"tag4", "value4"}
+                        }
+                };
+
+                var updateNamespaceResponse = ServiceBusManagementClient.Namespaces.Update(resourceGroup, namespaceName, updateNamespaceParameter);
+
+                TestUtilities.Wait(TimeSpan.FromSeconds(5));
+
+                // Get the updated namespace
+                getNamespaceResponse = ServiceBusManagementClient.Namespaces.Get(resourceGroup, namespaceName);
+                Assert.NotNull(getNamespaceResponse);
+                Assert.Equal(location, getNamespaceResponse.Location, StringComparer.CurrentCultureIgnoreCase);
+                Assert.Equal(namespaceName, getNamespaceResponse.Name);
+                Assert.Equal(2, getNamespaceResponse.Tags.Count);
+                foreach (var tag in updateNamespaceParameter.Tags)
+                {
+                    Assert.Contains(getNamespaceResponse.Tags, t => t.Key.Equals(tag.Key));
+                    Assert.Contains(getNamespaceResponse.Tags, t => t.Value.Equals(tag.Value));
+                }
+
+                //Update namespace sku capacity and tier to Premuium
+                var updateNamespaceCapacityParameter = new SBNamespaceUpdateParameters()
+                {
+                    Location = location,
+                    Sku = new SBSku
+                    {
+                        Capacity = 2,
+                        Name = SkuName.Premium,
+                        Tier = SkuTier.Premium
+                    },
+                    Tags = new Dictionary<string, string>()
+                        {
+                            {"tag3", "value3"},
+                            {"tag4", "value4"}
+                        }
+                };
+
+                var updateNamespaceCapacityResponse = ServiceBusManagementClient.Namespaces.Update(resourceGroup, namespaceName, updateNamespaceCapacityParameter);
+
+                TestUtilities.Wait(TimeSpan.FromSeconds(5));
+
+                // Get the updated namespace
+                getNamespaceResponse = ServiceBusManagementClient.Namespaces.Get(resourceGroup, namespaceName);
+                Assert.NotNull(getNamespaceResponse);
+                Assert.Equal(location, getNamespaceResponse.Location, StringComparer.CurrentCultureIgnoreCase);
+                Assert.Equal(namespaceName, getNamespaceResponse.Name);
+                Assert.Equal(2, getNamespaceResponse.Tags.Count);
+                Assert.Equal(updateNamespaceCapacityParameter.Sku.Name, getNamespaceResponse.Sku.Name);
+                Assert.Equal(updateNamespaceCapacityParameter.Sku.Tier, getNamespaceResponse.Sku.Tier);
+                Assert.Equal(updateNamespaceCapacityParameter.Sku.Capacity, getNamespaceResponse.Sku.Capacity);
+
+                //Update namespace sku tier to basic
+                var updateNamespaceSkuParameter = new SBNamespaceUpdateParameters()
```

Comment:
is the test executed with change SKU of namespace from Standard to Basic ?

Reply:
Tested from Standard to Basic without topic and with topic (handled conflict).